index.html

```diff
@@ -19,7 +21,7 @@ <h2>About:</h2>
     </section>
     <section>
       <h2>Experience:</h2>
-      <p>
+      <p id="experience">
```

Comment:
I think, for the `Experience` and `Skills` sections, they actually want styled blocks. You are headed in the right direction, but I think they want a bunch of items, which then "flex" wide or thin depending on the screen size, for this particular exercise. See this example:

https://github.com/ethomp-dev/intro-to-programming-eclipse/blob/main/examples/lesson-4-4/index.md